Sources/TidalAPI/generate_and_clean.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Define the input and output directories
+input_dir="Config/input"
+output_dir="Generated"
+
+# Step 1: Clear the contents of the input directory
+rm -rf "$input_dir"/*
+mkdir -p "$input_dir"
+
+# Step 2: Download the JSON files and save them in the input directory
+echo "Dowloading API specs from developer.tidal.com"
+curl -o "$input_dir/tidal-catalog-v2-openapi-3.0.json" https://developer.tidal.com/apiref/api-specifications/api-public-catalogue-jsonapi/tidal-catalog-v2-openapi-3.0.json
+curl -o "$input_dir/tidal-search-v2-openapi-3.0.json" https://developer.tidal.com/apiref/api-specifications/api-public-search-jsonapi/tidal-search-v2-openapi-3.0.json
+curl -o "$input_dir/tidal-user-v2-openapi-3.0.json" https://developer.tidal.com/apiref/api-specifications/api-public-user-jsonapi/tidal-user-v2-openapi-3.0.json
+curl -o "$input_dir/tidal-user-content-openapi-3.0.json" https://developer.tidal.com/apiref/api-specifications/api-public-user-content/tidal-user-content-openapi-3.0.json
+
+# Step 3: Clear the contents of the output directory
+echo "Clear the contents of the output directory"
+rm -rf "$output_dir"/*
+mkdir -p "$output_dir"
```

Comment:
Maybe this has been talked about already.

If we clean up the folder, and the new generated code has the same things as before, plus just one new endpoint (for example) would git detect everything as new, or would it detect just the new endpoint change? 

Would be great to have cleaner commits.

Reply:
> Would be great to have cleaner commits

well, we did most of the work in another (private) repo, so commits here can be too generic, I suppose 🙂 

Answering your question: it shows only changes.

Reply:
no no, I was not talking about the commits on this PR! 😁 

I was referring to the future commits when the generator updates things. But if it shows only changes, that is perfect! 👏